### PR TITLE
Alex first contribution

### PR DESCRIPTION
**Pull request by alexander-nitsche**
Hi Marvin,

this pull request is about some small beautifications that i did while starting to use your package for folder creation in my Neos 7 instance. They are about streamlining of wording and backend layout. Thanks for your efforts!

Greetings
Alex

### CODE REVIEW

**Comment by alexander-nitsche** [comment]
It looks like this on my Linux machine:
folder_backend_layout

**Comment by alexander-nitsche** [comment]
Added prefixed, unique CSS class names and a simplified, common template path. PR is ready for further processing.

**Review by breadlesscode — APPROVED** [review]
Awesome, thanks! ❤️ 

I will merge it soon

File: Resources/Private/Templates/Folder.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<div id="breadlesscode-node-types-folder">
```
Comment (breadlesscode) [inline Resources/Private/Templates/Folder.html:1]: Can we use AFX instead of Fluid?

**Review by breadlesscode — CHANGES_REQUESTED** [review]

File: Resources/Private/Templates/Folder.html (hunk 1 above)
Comment (alexander-nitsche) [reply Resources/Private/Templates/Folder.html:1]: Is AFX available for Neos 4?

Comment (breadlesscode) [reply Resources/Private/Templates/Folder.html:1]: Oh yes, your right. Thanks :)

**Pull request merged**